src/Conversion/FileConversionManager.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\Assets\Conversion;
+
+use SilverStripe\Assets\Storage\DBFile;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Injector\Injector;
+
+/**
+ * This class holds a list of available file converters which it uses to convert files from one format to another.
+ */
+class FileConversionManager
+{
+    use Configurable;
+
+    /**
+     * An array of classes or injector service names for
+     * classes that implement the FileConverter interface.
+     */
+    private static array $converters = [];
+
+    /**
+     * Convert the file to the given format using the first available converter that can perform the conversion.
+     *
+     * @param string $toFormat The file extension you want to convert to - e.g. "webp".
+     * @param array $options Any options defined for this converter which should apply to the conversion.
+     * Note that if a converter supports the conversion generally but doesn't support these options, that converter will not be used.
+     * @throws FileConversionException if the conversion failed or there were no converters available.
+     */
+    public function convert(DBFile $from, string $toFormat, array $options = []): DBFile
```

Comment:
Providing `$options` here gives a higher level of control for developers (and converters) which might require it.